R/utilities-resolution.R

```diff
@@ -33,5 +34,11 @@ resolution <- function(x, zero = TRUE, discrete = FALSE) {
   }
   d <- diff(sort(x))
   tolerance <- sqrt(.Machine$double.eps)
+
+  if(all(d < tolerance)){
+    return(1)
+  }
+
   min(d[d > tolerance])
+
```

Comment:
Thanks for this. I think this can be solved with fewer computations by doing something `min(d[d > tolerance], 1)`

Reply:
oh, I completely messed up in that suggestion. Sorry about that. 

You can revert to your old approach

Reply:
Ah, will fix